platform-includes/performance/automatic-instrumentation-custom-routing/javascript.mdx

```diff
@@ -0,0 +1,24 @@
+### Custom routing
+
+By default, the `browserTracingIntegration()` will create a `pageload` span for when the page is initially loaded, as well as a `navigation` span whenever the URL changes afterwards.
+
+If you want to ensure navigation spans are created correctly for a custom routing setup, you can opt out of the default `navigation` span creation by setting `instrumentNavigation: false` in the `browserTracingIntegration()` options. Then, you can manually create navigation spans like this:
+
+```javascript
+Sentry.init({
+  integrations: [
+    // disable default navigation spans
+    Sentry.browserTracingIntegration({ instrumentNavigation: false }),
+  ],
+});
+
+// Somewhere, instrument your router like this:
+myRouter.on("routeChange", (route) => {
```

Comment:
We should add a note about setting correct route name for pageload as well, we want to avoid un-paramaterized transactions.

Reply:
I guess this is mostly about updating the pageload span, right? 🤔 or should we document how to emit a pageload span?

Reply:
yeah updating the pageload span, I think most frameworks will be fine with emitting pageload as soon as possible.

Reply:
I opened a PR: https://github.com/getsentry/sentry-javascript/pull/10633 to make this easier, as documenting this showed that there is def. room for improvement there!